### DISCUSSION TITLE
Cannot upload/delete the specific folder via SFTP

### DISCUSSION THREAD

**Discussion by ss406022** [iterate-ch/cyberduck#15768] [Q&A]
Describe the problem
With the version of Cyberduck  8.4.2 (38090) I can't upload and deleted the specific folder via SFTP.
I want to update a file via SFTP using SSH private key and find that the file can't be upload. it shows the message "Upload failed. Permission denied. Please contact your web hosting service provider for assistance.", but there was no problem when I uploaded files to other folders.

Desktop
* OS: MacOS
* Version 13.6.5

**Comment by tatze96** [comment]
Please verify that you have sufficient permissions for the affected folder to write files to the folder or, respectively, delete the folder. Reach out to your server administrator to receive the corresponding information if necessary.

Also, does the error persist within the current release version (8.8.2) of Cyberduck?

If you have sufficient permissions and the error persists within the current release version [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the error message, and send us the log file afterward for further investigation.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
